docs/modules/ROOT/pages/documentation/payara-server/logging/exclude-fields.adoc

```diff
@@ -0,0 +1,51 @@
+[[exclude-fields]]
+= Exclude Fields
+
+All the three log formatters `ODLLogFormatter`, `UniformLogFormatter`, and `JSONLogFormatter` support the possibility to exclude some fields from the log entries.  This makes the log file more compact and removes unnecessary information in the case you do not need it or want to use it.
+
+
+You can change the `com.sun.enterprise.server.logging.GFFileHandler.excludeFields` within the _<PAYARA_HOME>/glassfish/domains/<domain-name>/config/logging.properties_ file or use the Admin Console or an `asadmin` command.
+
+When the configuration is changed when the server is running, a log rotation happens so that a log file always contains the same set of information.
```

Comment:
I would suggest:
If the configuration is changed while the server is running...